idb/postgres/setup_postgres_sql.go

```diff
@@ -52,6 +52,7 @@ CREATE TABLE IF NOT EXISTS account (
   microalgos bigint NOT NULL, -- okay because less than 2^54 Algos
   rewardsbase bigint NOT NULL,
   rewards_total bigint NOT NULL,
+  deleted boolean NOT NULL DEFAULT false,
```

Comment:
for got to add setup_postgres.sql to the PR?

Reply:
looks like it, fixed.